cclib/io/xyzwriter.py

```diff
@@ -91,6 +93,12 @@ def _xyz_from_ccdata(self, index):
             comment = "{}: Geometry {}".format(self.jobfilename, geometry_num)
         else:
             comment = "Geometry {}".format(geometry_num)
+        # Wrap the geometry number part of the comment in square brackets,
+        # prefixing it with one previously parsed if it existed.
+        if existing_comment:
+            comment = "{} [{}]".format(existing_comment, comment)
+        else:
+            comment = "[{}]".format(comment)
```

Comment:
For
> I think we could also make this more convenient by adding a comment argument into the XYZ class somewhere that does the writing. Internally using metadata to store the comment is fine, of course.

we could take this and make a new method out of it. Having a property on the class wouldn't make as much sense until we unify the readers and writers.